### ISSUE TITLE
Showing trendline for organization activity

### ISSUE THREAD

**Issue by rithviknishad** [coronasafe/leaderboard#250]
It'd be nice if we could visualize the growth of the community through the leaderboard.

We could show something similar to what GitHub shows for repository activity, but in the leaderboard's graph, we should show the activity at an organization level and not specific to a repository.

<img width="576" alt="image" src="https://github.com/coronasafe/leaderboard/assets/25143503/7e61ee6a-dc87-47f7-81ae-cf4174bf2563">

**Comment by kirtanchandak** [comment]
I would like to take this up!

**Comment by kirtanchandak** [comment]
I was thinking of showing graphs related to number of PRs, issues and commits in an organization over a period time period in a graph. What do you think?

**Comment by rithviknishad** [comment]
Yup! That'd be great.

We can also show the growth of the community as a separate graph too (maybe in `/people` page).

**Comment by PAVAN507-STAR** [comment]
@rithviknishad I would like to take up this issue.

**Comment by yuvraajnarula** [comment]
@rithviknishad 
I was interested in this issue. Could you please let me know if I can take up this issue?

**Comment by yuvraajnarula** [comment]
@rithviknishad 

![Image](https://github.com/user-attachments/assets/702a948f-453c-4614-b5a5-ac604d1e875c)

here is a graph for visualizing organization-level activity
I was thinking of making a for these 4 attributes in tabs to show different attributes vs time graphs. I wanted to know your opinion and also how would you like the contributor growth graph.

**Comment by yuvraajnarula** [comment]
@rithviknishad 
Any updates?

**Comment by rithviknishad** [comment]
yup, we can def. have this, but more importantly we need aggregate activity trendline graph too like shown on github.

**Comment by yuvraajnarula** [comment]
@rithviknishad 
I have made data collection via month-wise now to look much better from the UI perspective.
Also for the **aggregate trendline graph** I have tried to take every sort of interaction at the organizational level to showcase the activity
For contributors I have tried to show an increase graph to show growth from start to end 
Let me know if there is something I could do to change

https://github.com/user-attachments/assets/894bc5b0-17c2-41f9-95a9-56bca86f37a1

**Comment by yuvraajnarula** [comment]
@rithviknishad Any Updates?

**Comment by yuvraajnarula** [comment]
@rithviknishad Any updates?

**Comment by rithviknishad** [comment]
Yes, this looks great! Just confirming, 

1. Where is this placed? In `/leaderboard` page or Contributors Hall of Fame page?
2. If it's in `/leaderboard` page, (which would be preferred), let the time range of the chart be based on the date range filter specified in the leaderboard filters.

**Comment by yuvraajnarula** [comment]
this is placed in `/leaderboard`  page and yes I will do the changes of the time range part as per the filters specified and get back to you

**Comment by yuvraajnarula** [comment]
@rithviknishad 

https://github.com/user-attachments/assets/6b97eb7a-12d8-4562-88c0-366dc3b2e9f1

Any Changes for this

**Comment by yuvraajnarula** [comment]
@rithviknishad

**Comment by yuvraajnarula** [comment]
@rithviknishad 
Any Updates ?

**Comment by Yashgupta9330** [comment]
@rithviknishad, I would like to work on these issues. Can you assign them to me since I noticed there haven't been any updates for the past one and a half months? I will complete them within 2-3 days

**Comment by rithviknishad** [comment]
leaderboards architecture is being rewritten completely. since this is not adding a huge value by doing it right now, will reconsider assigning once that's completed.

**Comment by Yashgupta9330** [comment]
@rithviknishad Are all the issues listed in this repo not going to be assigned to anyone?

**Comment by rithviknishad** [comment]
mostly until rearchitecture is completed. trying to cleanup tech debts and make it more generic.

**Comment by rithviknishad** [comment]
leaderboard has been re-built, this is no longer valid. thanks for putting in efforts in trying to achieve to solve this!

the new leaderboard has achieved this.
